### PR TITLE
[Feature] Show floating notification badge in small views

### PR DESCRIPTION
**Pull request by sharkykh**
Just a small QoL thing...
Show a floating notification badge (for errors / warnings / news) on the navbar toggle button in when using small views such as mobile.

chrome_2018-03-26_20-47-10

### CODE REVIEW

File: medusa/server/web/core/base.py [hunk 1]
```diff
@@ -111,6 +111,7 @@ def __init__(self, rh, filename):
             'newsBadge': '',
             'toolsBadge': '',
             'toolsBadgeClass': '',
+            'floatingMenuBadge': '',
```
Comment (OmgImAlexis) [inline medusa/server/web/core/base.py]: No new python vars please. This should be implemented client side.

File: themes-default/legacy/static/css/style.css [hunk 2]
```diff
@@ -2843,6 +2843,20 @@ select,
     padding: 0;
 }
 
+.floating-badge {
```
Comment (OmgImAlexis) [inline themes-default/legacy/static/css/style.css]: No updating legacy please.

File: medusa/server/web/core/base.py (hunk 1 above)
Comment (sharkykh) [reply medusa/server/web/core/base.py]: Do you mean 'reading' the badge info from the tools badge and recreating it?

Comment (sharkykh) [reply medusa/server/web/core/base.py]: I should be more specific - most of the info needed to create a badge with the correct color is only server side (apart from the unread news count which is global (`app.UNREAD_NEWS`).

Comment (OmgImAlexis) [reply medusa/server/web/core/base.py]: I mean there should be no changes to the python code for any new ui changes. Everything should now be done client side. The badges and HTML that're currently handled server side like the ones above will all be removed very soon.

**Pull request closed without merging**